demo/src/features/sprite/prepare.rs

```diff
@@ -120,13 +120,13 @@ impl PrepareJob for SpritePrepareJob {
         let aspect_ratio = extents_width as f32 / extents_height as f32;
         let half_width = 400.0;
         let half_height = 400.0 / aspect_ratio;
-        let view_proj = glam::Mat4::orthographic_rh_gl(
+        let view_proj = glam::Mat4::orthographic_rh(
             -half_width,
             half_width,
             -half_height,
             half_height,
-            -100.0,
-            100.0,
+            1000.0,
```

Comment:
we're using reverse z, so flip the sign here. I also think for sprite rendering, depth: [-1000,1000] is plenty of precision and should be enough for most people